javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/script.js

```diff
@@ -1,384 +1,265 @@
- //  Songs array creation
- const songs = [
+const playlistSongs = document.getElementById("playlist-songs");
+const playButton = document.getElementById("play");
+const pauseButton = document.getElementById("pause");
+const nextButton = document.getElementById("next");
+const previousButton = document.getElementById("previous");
+const shuffleButton = document.getElementById("shuffle");
+
+const cover =
+  "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg";
+const allSongs = [
   {
-    id:"0",
+    id: "0",
     title: "Scratching The Surface",
-    cover:
-      "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg",
+    cover,
```

Comment:
If we are removing the cover variable, then all of these referencing to it can be removed too